user_guide_src/source/testing/controllers/001.php

```diff
@@ -1,12 +1,12 @@
 <?php
 
-namespace CodeIgniter;
+namespace App\Controllers;
```

Comment:
Why do we want to use `App\Controllers` namespace for the tests?

Reply:
In my understanding, in the docs, we use the same namespace as the test target, in this case a Controller.
See https://www.codeigniter.com/user_guide/testing/overview.html#the-test-class

Reply:
Fair enough. It seems a bit odd, but we can follow the user guide.

Reply:
You feel a bit odd, but I think this change is consistent.
And if you have better convention, I don't mind to change the entire tests namespacing.

Reply:
> You feel a bit odd, but I think this change is consistent.

Yes, of course. This change is consistent with what we already have.

> And if you have better convention, I don't mind to change the entire tests namespacing.

It is only about the cases when we create tests for our own application. It's strange that we have such a convention for namespace. I would have expected the use of the `Tests` namespace, like `Tests\Controllers` etc...

If you would like to make such changes, I would certainly accept them, but I don't want it to look like I'm trying to force it.